src/data/graphics/gimmicks.h

```diff
@@ -103,7 +103,7 @@ static const struct SpriteSheet sSpriteSheet_MegaIndicator = {sMegaIndicatorGfx,
 static const struct SpriteSheet sSpriteSheet_AlphaIndicator = {sAlphaIndicatorGfx, sizeof(sAlphaIndicatorGfx), TAG_ALPHA_INDICATOR_TILE};
 static const struct SpriteSheet sSpriteSheet_OmegaIndicator = {sOmegaIndicatorGfx, sizeof(sOmegaIndicatorGfx), TAG_OMEGA_INDICATOR_TILE};
 static const struct SpriteSheet sSpriteSheet_DynamaxIndicator = {sDynamaxIndicatorGfx, sizeof(sDynamaxIndicatorGfx), TAG_DYNAMAX_INDICATOR_TILE};
-static const struct SpriteSheet sTeraIndicatorSpriteSheets[NUMBER_OF_MON_TYPES + 1] =
+static const struct SpriteSheet sTeraIndicatorSpriteSheets[NUMBER_OF_MON_TYPES] =
 {
     {sNormalIndicatorGfx, sizeof(sNormalIndicatorGfx), TAG_NORMAL_INDICATOR_TILE},
```

Comment:
shouldn't it be solved by adding a dummy in this array as the first entry?

Reply:
I tried, but for some reason that absolutely shifted everything (it even started to look like the mega evolution icon)

Reply:
It could be that I am misunderstanding the issue though since I don't understand how decreasing it in `sTeraIndicatorSpriteSheets[GetBattlerTeraType(battler) - 1]` solves the problem.

Reply:
> I tried, but for some reason that absolutely shifted everything (it even started to look like the mega evolution icon)

Interesting. No idea why